articles/client-platforms/_includes/_authorization-create-rule.md

```diff
@@ -0,0 +1,9 @@
+First, we will create a rule that assigns our users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](https://manage.auth0.com/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
```

Comment:
@flaviogranero 
Please update `[new rule page](https://manage.auth0.com/#/rules/new)` to `[new rule page](${uiURL}/#/rules/new)`